src/core/ugen_stk.cpp

```diff
@@ -8486,7 +8486,7 @@ void Echo :: set( MY_FLOAT max )
     MY_FLOAT delay = delayLine ? delayLine->getDelay() : length>>1;
     if( delayLine ) delete delayLine;
     if( delay >= max ) delay = max;
-    delayLine = new Delay(length>>1, length);
+    delayLine = new class Delay(length>>1, length);
     this->clear();
```

Comment:
global namespace pollution fix: `Delay` is some global function in UE on macOS, a C++ language feature allows us to disambiguate with the `class` specifier before usages of the class